src/config.ts

```diff
@@ -0,0 +1,51 @@
+import fs from "node:fs";
+import path from "node:path";
+import json5 from "json5";
+
+export interface Config {
+  readonly ignored?: readonly string[];
+}
+
+const CONFIG_FILENAME = "pnpm-deduplicate.json";
+
+export async function readConfig(dir: string): Promise<Config | null> {
+  const configPath = path.join(dir, CONFIG_FILENAME);
+
+  let buf: Buffer;
+  try {
+    buf = await fs.promises.readFile(configPath);
+  } catch (err: unknown) {
+    if ((err as { code?: string }).code === "ENOENT") {
+      return null;
+    }
+    throw err;
+  }
+
+  const parsed = json5.parse(buf.toString());
```

Comment:
Using `json5` to allow comments in the new config file. I'd like to write the reason some packages are excluded from the fixer/check.